plugins/com.google.cloud.tools.eclipse.appengine.compat.test/.classpath

```diff
@@ -2,6 +2,10 @@
 <classpath>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="src/">
+		<attributes>
+			<attribute name="test" value="true"/>
```

Comment:
Probably we should revert these changes. Having the "test" attribute for "src/" doesn't seem right.

Reply:
Oh!  I had marked those changes to be ignored — Eclipse 2018-09 or -12 added support for app vs test scoping of dependencies.